kolibri/plugins/facility/assets/src/views/FacilityConfigPage/index.vue

```diff
@@ -218,23 +216,30 @@
           this.toggleSetting('learner_can_edit_password');
         }
       },
-      dismissNotification() {
-        this.$store.commit('facilityConfig/CONFIG_PAGE_NOTIFY', null);
+      updateSettings(action) {
+        this.$store
+          .dispatch(action)
+          .then(() => {
+            // eslint-disable-next-line kolibri/vue-no-undefined-string-uses
+            this.createSnackbar(this.NotificationsStrings.$tr('saveSuccess'));
+            this.copySettings();
+          })
+          .catch(() => {
+            // eslint-disable-next-line kolibri/vue-no-undefined-string-uses
+            this.createSnackbar(this.NotificationsStrings.$tr('saveFailure'));
+            this.$store.commit('CONFIG_PAGE_UNDO_SETTINGS_CHANGE');
+          });
       },
       resetToDefaultSettings() {
         this.showModal = false;
-        this.$store.dispatch('facilityConfig/resetFacilityConfig').then(() => {
-          this.copySettings();
```

Comment:
We need to keep this call to `copySettings` somewhere inside of `updateSettings`. This creates a copy of the new settings object in order to do a "dirty check" if the user changes things again after saving.